api/src/utils/get-local-type.ts

```diff
@@ -128,6 +127,11 @@ export default function getLocalType(
 		return 'text';
 	}
 
+	/** Handle Boolean as TINYINT*/
+	if (column.data_type === 'tinyint(1)' || column.data_type === 'tinyint(0)') {
```

Comment:
Lets add in a `toLowerCase()` here just in case some database wants to throw you a curveball with `TINYINT` instead of `tinyint`